### DISCUSSION TITLE
Add a back button on File/Folder Selector

### DISCUSSION THREAD

**Discussion by honguyenminh** [ppy/osu#15639] [Ideas]
Add a back button on File/Folder Selector interface to go back to where we were last on.
Currently there's no way to go back to where we was before changing folder. Especially on Android (11), when you open root folder accidentally, you'll get access denied on everything and can't go back to `/sdcard` too (this folder itself is another reported issue but let's set that aside for now). Users have to get out of the selector, open the selector again then go back to where they're last in.

This might need some change to framework though.

**Comment by peppy** [comment]
An alternative might be to not change folders when the result is an irrecoverable screen (ie. access denied), showing an error instead.

**Reply by honguyenminh** [reply]
Then it should be easy to implement, just return when `notifyError` is true and only clear the file list afterward, not before checking for permission like the current code.

But this change would need to be in osu framework though.
